<ui version="4.0" >
 <class>magnatuneDownloadDialogBase</class>
 <widget class="QDialog" name="magnatuneDownloadDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Magnatune.com Album Download</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Download options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Select Format:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="formatComboBox" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>Download to:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KUrlRequester" name="downloadTargetURLRequester" />
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>If you download to a location that is already being monitored by Amarok, the album will automatically be added to your collection.</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Magnatune info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QTextEdit" name="infoEdit" >
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="downloadButton" >
     <property name="text" >
      <string>&amp;Download</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>magnatuneDownloadDialogBase</receiver>
   <slot>downloadButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel" >
     <x>262</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
